packages/vite-node/src/cli.ts

```diff
@@ -12,6 +13,9 @@ cli
   .option('-r, --root <path>', 'Use specified root directory')
   .option('-c, --config <path>', 'Use specified config file')
   .option('-w, --watch', 'Restart on file changes, similar to "nodemon"')
+  // TODO: How could we document this? Ideally, we should link to the
+  // TODO: ViteNodeServerOptions type since it's the source of truth.
+  .option('--server-options <options>', 'Use specified Vite server options')
```

Comment:
I could not find any proper example of such documentation. Should I add a few lines to the README?

Reply:
Yeah I think that would do